src/main/java/org/usfirst/frc/team2473/robot/commands/ElevatorTicksTest.java

```diff
@@ -0,0 +1,51 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team2473.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.usfirst.frc.team2473.robot.Robot;
+
+import java.io.IOException;
+
+import org.usfirst.frc.team2473.framework.Devices;
+import org.usfirst.frc.team2473.framework.UtilitySocket;
+
+/**
+ * A class that aligns the robot to the hatch based on the angle provided by CV
```

Comment:
https://youtu.be/BLikP6BDH5w

Reply:
Nice 6 second YouTube video! Commented it out